addons/sale_project/models/sale_order_line.py

```diff
@@ -173,6 +173,13 @@ def write(self, values):
                     line.task_id.write({'allocated_hours': allocated_hours})
         return result
 
+    def copy_data(self, default=None):
+        data = super().copy_data(default)
+        for origin, datum in zip(self, data):
+            if origin.analytic_distribution == origin.order_id.project_id._get_analytic_distribution():
```

Comment:
I think it could crash if the user does not have access to the project.

```suggestion
            if origin.analytic_distribution == origin.order_id.project_id.sudo()._get_analytic_distribution():
```

what do you think? 🤔